ckanext/datastore/backend/__init__.py

```diff
@@ -108,7 +108,11 @@ def configure(self, config: CKANConfig):
 
         return config
 
-    def create(self, context: Context, data_dict: dict[str, Any]) -> Any:
+    def create(
+            self,
+            context: Context,
+            data_dict: dict[str, Any],
+            plugin_data: dict[int, dict[str, Any]]) -> Any:
```

Comment:
I did consider that. AFAIK there are no active users of IDatastoreBackend, and I dislike function signatures that give no information about required parameters and their types. This kind of pattern across our code base makes it difficult to reason about code without reading the implementation, and encourages bad behaviours like passing the same context and data_dict through many layers of code and each layer often mutating the dict before returning.

So in this case I resisted :-) a better fix in my mind is to break out the current parameters but that was a larger change. I'm happy to add it to data_dict here if you think this will cause unnecessary breakage of existing code.